goi18n/testdata/expected/R.go

```diff
@@ -0,0 +1,26 @@
+// DON'T CHANGE THIS FILE MANUALLY
+// This file was generate using the command:
+// $ goi18n constants
+
+package R
+
+// DDays id:d_days
+const DDays string = "d_days"
+
+// MyHeightInMeters id:my_height_in_meters
+const MyHeightInMeters string = "my_height_in_meters"
```

Comment:
Is there a reason you are explicitly specifying the string type? I would just omit it.

```
const MyHeightInMeters = "my_height_in_meters"
```